### ISSUE TITLE
ReadTheDocs builds failing

### ISSUE THREAD

**Issue by pao** [JuliaLang/julia#11188]
First failing build: https://readthedocs.org/builds/julia/2708803/ (at 1df9c1a137049862dff7d9d1e9a46fd84fcc097a)

Error message:
```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/julia/envs/latest/local/lib/python2.7/site-packages/sphinx/cmdline.py", line 244, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/julia/envs/latest/local/lib/python2.7/site-packages/sphinx/application.py", line 143, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/julia/envs/latest/local/lib/python2.7/site-packages/sphinx/application.py", line 433, in setup_extension
    err)
ExtensionError: Could not import extension juliadoc.jldoctest (exception: cannot import name bytes)

Extension error:
Could not import extension juliadoc.jldoctest (exception: cannot import name bytes)
```

possible lead: http://stackoverflow.com/questions/27405405/i-get-an-importerror-cannot-import-name-bytes-with-pygments

cf dicscussion at end of #11099

**Comment by pao** [comment]
https://github.com/JuliaLang/JuliaDoc/blob/master/juliadoc/jldoctest.py#L29

**Comment by pao** [comment]
Should be fixable if there's an alternative for this usage:

https://github.com/JuliaLang/JuliaDoc/blob/master/juliadoc/jldoctest.py#L693-L694

**Comment by pao** [comment]
RTD upgraded from Sphinx 1.2.2 to 1.3.1 between the last successful and first failing build.

**Comment by pao** [comment]
1.3b3 release notes: "sphinx.util.pycompat has been restored in its backwards-compatibility; slated for removal in Sphinx 1.4."

Clearly that didn't go entirely as planned.

**Comment by pao** [comment]
Should be fixed by 47e91ea, which in turn pulls https://github.com/JuliaLang/JuliaDoc/commit/4ea9c645f353c260439c0a4d18ad1e48879c83af.
